openjdk/openjdk.mk

```diff
@@ -121,9 +121,7 @@ CUSTOM_NATIVE_OPTIONS :=
 
 ifneq ($(JDK_VERSION),8)
 	ifdef TESTIMAGE_PATH
-		ifneq ($(OS),OS/390)
-			JDK_NATIVE_OPTIONS := -nativepath:"$(TESTIMAGE_PATH)$(D)jdk$(D)jtreg$(D)native"
-		endif
+		JDK_NATIVE_OPTIONS := -nativepath:"$(TESTIMAGE_PATH)$(D)jdk$(D)jtreg$(D)native"
```

Comment:
This is the initial setup before the https://github.com/adoptium/aqa-tests/pull/1707/files. Does this mean zos jdk has testimage available now?

Reply:
Yes sophia, we've jdk test image for z/OS now. [This is the PR](https://github.com/eclipse-openj9/openj9/pull/13073) which had enabled jdk natives on z/OS